docs/src/comparison.md

```diff
@@ -12,24 +11,13 @@ We used the following models for the comparison:
 
-Time taken is presented in normalised units, measured against the runtime of Agents.jl. In other words: the results can only vary slightly from the ones presented here with a different hardware. If one wishes to repeat the results personally by using the scripts in the [ABM_Framework_Comparisons](https://github.com/JuliaDynamics/ABM_Framework_Comparisons) repository: they will compute nearly the same results. For details on the parameters used for each comparison and the hardware used for the latest run, see the `README` and the `benchmark.jl` file in that repository.
+Time taken is presented in normalised units, measured against the runtime of Agents.jl. In other words: the results can only vary slightly from the ones presented here with a different hardware.
 
 For LOC, we use the following convention: code is formatted using standard practices & linting for the associated language. Documentation strings and in-line comments (residing on lines of their own) are discarded, as well as any benchmark infrastructure. NetLogo is assigned two values since its files have a code base section and an encoding of the GUI. Since many parameters live in the GUI, we must take this into account. Thus `375 (785)` in a NetLogo count means 375 lines in the code section, 785 lines total in the file. An additional complication to this value in NetLogo is that it stores plotting information (colours, shapes, sizes) as agent properties, and as such the number outside of the bracket may be slightly inflated.
 
-| Model/Framework | Agents 5.12.0 | Mesa 1.2.0 | Netlogo 6.3.0 | MASON 21.0 |
-|---|---|---|---|---|
-|Wolf Sheep Grass|1|50.2x|28.2x|NA|
-|(LOC)|122|227|137 (871)| . |
-|Flocking|1|129.8x|25.5xᕯ|9.0x|
-|(LOC)|62|102|82 (689)|369|
-|Forest Fire|1|303.7x|222.2x|NA|
-|(LOC)|23|35|43 (545)|.|
-|Schelling|1|209.4x|166.4x|195.9x|
-|(LOC)|31|56|60 (743)|248|
+You can find the latest results in the `README.md` of the [ABM_Framework_Comparisons](https://github.com/JuliaDynamics/ABM_Framework_Comparisons) repository where you can find also the details on the parameters used for each comparison in the `DECLARATION.md` files inside each model subfolder. 
 
-ᕯ Netlogo has a different implementation to the other three frameworks here. It cheats a little by only choosing one nearest neighbor in some cases rather than considering all neighbors within vision. So a true comparison would ultimately see a slower result.
-
-The results clearly speak for themselves. Across all four models, Agents.jl's performance is exceptional whilst using the least amount of code. This removes many frustrating barriers-to-entry for new users, and streamlines the development process for established ones.
+Across all four models, Agents.jl's performance is exceptional whilst using the least amount of code. This removes many frustrating barriers-to-entry for new users, and streamlines the development process for established ones.
```

Comment:
Instead of saying "across all four models", we should write "In the majority of cases". This allows the possibility that there could be a case where this isn't true in the future. It also allows more models.

Reply:
indeed also there are just three models now :P, but I agree on saying something less strict